src/components/PopOver/index.js

```diff
@@ -135,17 +136,22 @@ class PopOver extends Component {
         ? topPosition
         : bottomPosition;
 
-    return {
-      x: Math.min(
-        Math.max(elLeft + elWidth / 2 - width / 2, containerLeft),
-        windowWidth - spaceFromEdge - width,
-        containerRight
-      ),
-      y:
-        bottomPosition + height + spaceFromEdge <=
+    const getXPosition = Math.min(
+      Math.max(elLeft + elWidth / 2 - width / 2, containerLeft),
+      windowWidth - spaceFromEdge - width,
+      containerRight
+    );
+
+    const getYPosition =
```

Comment:
Rename the variables to remove the get prefix

Reply:
corrected